glean-core/python/glean/metrics/labeled.py

```diff
@@ -3,7 +3,7 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
-from typing import Any, Optional, Set, Type
+from typing import Any, Optional, Set, Type, Dict
```

Comment:
```suggestion
from typing import Any, Dict, Optional, Set, Type
```

nit, keeping things alphabetized